src/main/java/io/supertokens/webserver/api/jwt/JWTSigningAPI.java

```diff
@@ -68,7 +69,7 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
                     new WebserverAPI.BadRequestException("validity must be greater than or equal to 0"));
         }
 
-        boolean useDynamicKey = version.equals("2.18") &&
+        boolean useDynamicKey = version.equals(SemVer.v2_19) &&
```

Comment:
using equals is still not the correct thing? We need to use version.greaterThanEquals?